nitrite.go

```diff
@@ -189,8 +189,24 @@ type CertProvider interface {
 	Roots() (*x509.CertPool, error)
 }
 
-var NewNitroCertProvider = internal.NewNitroCertProvider
-var NewSelfSignedCertProvider = internal.NewSelfSignedCertProvider
+func NewNitroCertProvider() CertProvider {
```

Comment:
Consider changing to `NewEmbeddedNitroCertProvider` to match format followed by `NewFetchingNitroCertProvider`

Reply:
done